src/test/java/nextstep/subway/acceptance/PathSteps.java

```diff
@@ -0,0 +1,27 @@
+package nextstep.subway.acceptance;
+
+import io.restassured.RestAssured;
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
+import io.restassured.specification.RequestSpecification;
+import org.springframework.http.MediaType;
+
+public class PathSteps {
+    public static ExtractableResponse<Response> 두_역의_최단_거리_경로_조회를_요청(Long source, Long target) {
+        return 두_역의_경로_조회_요청(RestAssured.given(), source, target, "DISTANCE");
+    }
+
+    public static ExtractableResponse<Response> 두_역의_최소_시간_경로_조회를_요청(Long source, Long target) {
+        return 두_역의_경로_조회_요청(RestAssured.given(), source, target, "DURATION");
+    }
+
+    public static ExtractableResponse<Response> 두_역의_경로_조회_요청(RequestSpecification given, Long source, Long target, String type) {
+        return given.log().all()
```

Comment:
```suggestion
        return RestAssured.given().log().all()
```
이렇게 써야 중복을 줄일 수 있지 않을까요?